tests/python/contrib/test_cmsisnn/test_conv2d.py

```diff
@@ -129,8 +128,11 @@ def make_model(
 @pytest.mark.parametrize("kernel_size", [(3, 3)])
 @pytest.mark.parametrize("padding", ["SAME", "VALID"])
 @pytest.mark.parametrize("strides, dilation", [((2, 2), (1, 1)), ((1, 1), (1, 1))])
-@pytest.mark.parametrize("enable_bias", [True, False])
 @pytest.mark.parametrize("relu_type", ["NONE", "RELU"])
+@pytest.mark.parametrize(
+    "conv_type, depth_multiplier, enable_bias",
+    [("conv2d", 1, True), ("conv2d", 1, False), ("depthwise", 1, True), ("depthwise", 3, True)],
```

Comment:
Do we need to run all the tests again for a different depth_multiplier ?

Reply:
The problem is depth_multipler has to be 1 in Conv2D. Depthwise support 1 and non-1 depth multipliers both. Earlier I'd kept it as a separate parameter.

Reply:
From out offline discussion, since fvp takes a lot of time, it makes sense to reduce number of tests for now.